src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+}
+
+.search-form-small {
+  height: 42px;
+}
+
+.search-input {
+  width: 100%;
+  height: 100%;
+  padding: 0 0 0 62px;
+  background: url(./images/Search.svg) no-repeat;
```

Comment:
background image disappears in focus state, you should style it as a` after` element of the label  and position it `absolute`.